Sections/Dashboard.cs

```diff
@@ -1,18 +1,25 @@
+using System.Collections.ObjectModel;
+using SimHub.Plugins.OutputPlugins.GraphicalDash;
+
 namespace DahlDesign.Plugin.Categories
 {
     public class Dashboard
     {
         private readonly DahlDesign Base;
 
- 
+        public Screen LeftScreen;
+        public Screen RightScreen;
         public Dashboard(DahlDesign dahlDesign)
         {
             Base = dahlDesign;
 
+            LeftScreen = new Screen(new string[] { "TimeAttack", "Sector", "LapTime", "Quali", "Race", "Race2", "Race3", "ShiftPoint", "Map" });
+            RightScreen = new Screen(new string[] { "Practice", "Stint", "Q1", "Q2", "Race1", "Race2", "Ralley", "Track" });
```

Comment:
We should change the name of these pages:
- Stint1
- Stint2
- Qualy1
- Qualy2
- Race1
- Race2
- Race3
- Track